src/tracing_utils.rs

```diff
@@ -0,0 +1,9 @@
+/// The resource representing a registered subscriber for non_blocking tracing logger
```

Comment:
We will be adding more logging, monitoring functionality to this repo in the future(and need more utility functions), so I thought it would be better to create that file instead of have the struct definition be inside main.rs